docs/connector/08-03-revoke-client-certificate.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Revoke client certificate
+type: Tutorials
+---
+
+After you have established a secure connection with Compass and generated a client certificate, you may want to revoke this certificate at some point. To revoke a client certificate, follow the steps in this tutorial.
+
+> **NOTE:** A revoked client certificate remains valid until it expires, but it cannot be renewed.
+
+## Prerequisites
+
+- [OpenSSL toolkit](https://www.openssl.org/docs/man1.0.2/apps/openssl.html) to create a Certificate Signing Request (CSR), keys, and certificates which meet high security standards
+- [Established secure connection with Compass](08-01-establish-secure-connection-with-compass.md)
```

Comment:
Maybe this prerequisite should be the last one on the list, as the user must have Compass, Application, and Runtime first.